src/components/intro/index.tsx

```diff
@@ -47,11 +47,27 @@ interface Props {
 }
 interface State {
   fetchingDemoIndex?: number;
+  deferredPrompt?: BeforeInstallPromptEvent;
+  installSource?: String;
 }
 
 export default class Intro extends Component<Props, State> {
-  state: State = {};
+  state: State = {
+    deferredPrompt: undefined,
+    installSource: undefined,
+  };
```

Comment:
```suggestion
  state: State = {};
```

`undefined` is implicit :D